main.tex

```diff
@@ -347,14 +347,45 @@ \subsection{Support of Astropy}
 Different funding models have been proposed for support of \astropy
 (e.g., \citealt{2016arXiv161003159M}), but a long-term plan
 for sustainability has not yet been established.
-The \astropy project has the ability to accept financial contributions
+The \astropy Project has the ability to accept financial contributions
 from institutions or individuals through the NumFOCUS\footnote{NumFOCUS
 is a 501(c)(3) nonprofit that supports and promotes world-class, innovative,
-open source scientific computing.} organization. NumFOCUS has, to date, covered the direct costs incurred by the \astropy project.
+open source scientific computing.} organization. NumFOCUS has, to date, covered the direct costs incurred by the \astropy Project.
+
+\subsection{Reuse of the scientific Python ecosystem}
+
+The \astropy Project is built on a philosophy of building from the existing
+Python scientific ecosystem wherever possible. Many of the enabling technologies
+like \texttt{numpy}, \texttt{scipy}, \texttt{matplotlib}, or \texttt{cython},
+are necessary as the core underpinnings of \astropy packages.  Often this means
+using these packages as ``building blocks'' (e.g., the ubiquitous
+use of \texttt{numpy} arrays throughout \astropypkg).  In other cases this means
+wrappers around more general algorithms that make them more convenient for
+astronomy use cases (e.g., the model-fitting in the \package{astropy.modeling}
+subpackage discussed in \sectionname~\ref{sec:modeling}).  Sometimes these
+simple wrappers evolve into more complex implementations that address
+astronomically-relevant use cases the general tool does not support (e.g.,
+\package{astropy.convolution}, see \sectionname~\ref{ssec:convolution}). As a
+broad rule, the Project explicitly encourages re-use of code where possible.
+
+However, the boundaries of when this re-use is called for often ambiguous.  Some
+of the examples above only evolved after significant debate in the community
+over whether these algorithms were sufficient.  Other times, even apparently
+general functionality did not exist in the wider ecosystem that met the \astropy
+community's needs, and hence the functionality had to be developed wholly from
+the developer resources available in the community (e.g.,
+\package{astropy.units}, or \package{astropy.table} when it began). While some
+of this functionality is ``given back'' to the wider community (e.g., myriad bug
+fixes for the enabling technologies listed above, and some of the testing and
+documentation architecture), in general, careful consideration is taken of
+whether the additional effort to make a piece of \astropy, and it is only
+``extracted'' when it seems plausible that the ongoing effort is worth the
+reward in the form of additional contributors or maintainers.
+
```

Comment:
Suggested replacement to split up the sentence starting from 'While some of this...'

To the wider community, the \astropy Project has provided myriad bug fixes to the enabling technologies above as well to the testing and documentation architecture.    Functionality is only 'extracted' from \astropy to other packages after careful consideration that includes considering the impact on the maintenance and support of \astropy.

Reply:
I like the new suggestion.